monero-outreach-docs/translations/pt-br/storyboard/tool_for_renewal_pt_br.md

```diff
@@ -0,0 +1,77 @@
+
+Contudo, sendo novo, Bitcoin teve falhas técnicas, incluindo pouca privacidade. E então outro personagem ou grupo pseudônimo atendido pelo nome de Nicolas van Saberhagen desenvolveu o protocolo CryptoNote. Desenvolvedores tomaram conhecimento e assim nasceu o Monero. Monero resolveu vários problemas do Bitcoin, incluindo ser facilmente vigiado por companhias e governos. Empresas de analise de blockchain como a Chainalysis declarou recentemente em um webinar público, [chainalysis.com/.../Advanced-Obfuscation-Techniques-Public-Webinar.mp4](https://go.chainalysis.com/rs/503-FAP-074/images/Advanced-Obfuscation-Techniques-Public-Webinar.mp4), “Monero é muito mais focada em privacidade do que seus similares… Tudo no Monero é ofuscado, você não pode distinguir nada."
+
+Monero é totalmente código aberto e tem uma das mais ativas e disseminadas comunidades de quaisquer criptomoedas. A comunidade tem ganhado reputação para um ethos que suporta autodeterminação, soberania, e a proteção dos direitos humanos. Se você está procurando por um sistema que pode tirar pode de governos corruptos e banqueiros centrais - e ultimamente melhorar como as pessoas interagem e transacionam umas com as outras – nós convidamos você a usar Monero.
+
+Quando uma transação é realizada usando Monero, o remetente, o receptor, e a quantia são todos ocultados usando criptografia, prevenindo assim que governos, empresas, e outros, vejam e gravem suas ações. Com Monero, compradores e vendedores individuais obtém a proteção à privacidade que se espera para atividades pessoas em sua própria casa. Matemática e software de código aberto protegem você, sem bancos, sistema legal, ou a polícia. 
+
+Se você precisa mover seu dinheiro entre uma reserva de valor privada por necessidade ou você deseja participar a princípio para optar por não participar do sistema financeiro atual, junte-se a nós. Juntos, nós podemos usar novas tecnologias para limitar a intrusão banqueiros e governos corruptos, e participar de um sistema financeiro que capacita as pessoas. Estamos todos juntos nisso. Vamos usar estas ferramentas para sermos a mudança que desejamos ver no mundo, por nós e pelas futuras gerações.
+
+Nós não esperamos que você aceite nossas palavras por nada disso e encorajamos você a fazer sua própria pesquisa e verificar com fontes. Nós compartilharemos alguns pontos iniciais aqui. Concluindo sua pesquisa, por favor volte para o guia rápido sobre como iniciar usando Monero (veja abaixo).
+
+### _Leitura Complementar:_
```

Comment:
The Monero and Outreach websites have many articles already translated to Portuguese. I suggest you to check if the links in this text have the Portuguese version too.

Reply:
About the links, this is what botoserelepe meant. There are already portuguese links we could use inside the translation.